apps/backtalk/src/components/backtalk/results/SurveyResults.tsx

```diff
@@ -112,7 +112,7 @@ export const SurveyResults: FC<Props> = ({ host }) => {
     [data?.surveys_by_pk?.id],
   );
 
-  const { hasCopied, onCopy } = useClipboard(surveyLink);
+  const { hasCopied, onCopy } = useClipboard(`${host}`+surveyLink);
```

Comment:
I think it would be a good idea to append the protocol to the front of this. This is so that when it gets copied and pasted it will be clear that it's an HTTP link, and also in some more rudimentary implementations of checking for links, it won't recognize it without a protocol prefix.